web/pdf_attachment_viewer.js

```diff
@@ -89,6 +89,43 @@ var PDFAttachmentViewer = (function PDFAttachmentViewerClosure() {
     /**
      * @private
      */
+    _bindPdfLink:
+        function PDFAttachmentViewer_bindPdfLink(button, content, filename) {
+      var blobUrl;
+      button.onclick = function() {
+        if (!blobUrl) {
+          blobUrl = pdfjsLib.createObjectURL(
+            content, 'application/pdf', pdfjsLib.PDFJS.disableCreateObjectURL);
+        }
+        var viewerUrl;
+        if (typeof PDFJSDev === 'undefined' || PDFJSDev.test('GENERIC')) {
+          // The current URL is the viewer, let's use it and append the file.
+          viewerUrl = '?file=' + encodeURIComponent(blobUrl + '#' + filename);
+        } else if (PDFJSDev.test('CHROME')) {
+          // In the Chrome extension, the URL is rewritten using the history API
+          // in viewer.js, so an absolute URL must be generated.
+          // eslint-disable-next-line no-undef
+          viewerUrl = chrome.runtime.getURL('/content/web/viewer.html') +
+            '?file=' + encodeURIComponent(blobUrl + '#' + filename);
+        } else {
```

Comment:
Done. I initially kept them separate in case someone ever has a need for disabling `createObjectURL` in Firefox (and wants to be able to revert the commit), but now I have squashed the two commits with an extra comment and updated commit message.